src/core/Flora/Model/Release/Types.hs

```diff
@@ -88,6 +89,7 @@ data Release = Release
   , readmeStatus :: ImportStatus
   , changelog :: Maybe TextHtml
   , changelogStatus :: ImportStatus
+  , tarball :: Maybe Sha256Sum
```

Comment:
```suggestion
  , tarballHash :: Maybe Sha256Sum
```

Reply:
Also I believe you'll need a migration to add this field to the releases table?

Reply:
It's [here](https://github.com/flora-pm/flora-server/pull/452/files#diff-f9b522a0d4932f5c0ac27448a28fbfa1728086fb5618382efe62f883b7d37fd9R11), tests wouldn't have passed otherwise!